src/Uno.UI/UI/Xaml/Controls/ItemsStackPanel/ItemsStackPanel.cs

```diff
@@ -58,6 +62,43 @@ private void CreateLayoutIfNeeded()
 #endif
 			}
 		}
+
+		void IInsertionPanel.GetInsertionIndexes(Windows.Foundation.Point position, out int first, out int second)
+		{
+			first = -1;
+			second = -1;
+			if ((new Windows.Foundation.Rect(0, 0, ActualSize.X, ActualSize.Y)).Contains(position))
```

Comment:
I think there's something slightly more efficient...
```suggestion
			if ((new Windows.Foundation.Rect(default, ActualSize)).Contains(position))
```